### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by maccybee**
[createPayment.csv.xlsx](https://github.com/Greeff/tech_test_perf/files/773667/createPayment.csv.xlsx)
[username.csv.xlsx](https://github.com/Greeff/tech_test_perf/files/773666/username.csv.xlsx)
[PerfTestTest_01.zip](https://github.com/Greeff/tech_test_perf/files/773719/PerfTestTest_01.zip)

### CODE REVIEW

**Comment by Greeff** [comment]
@maccybee why was the pull request closed?

**Comment by maccybee** [comment]
Idiot move on my part!

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+*******************READ ME*******************
+
+
+* The exercise has been completed in Jmeter, simulated against a local apache server (localhost, port 8080)
+
+* The folder contains the .jmx file (open Jmeter first and then open the project file); a CSV for user login and a CSV for payment info
```
Comment (Greeff) [inline readme.txt]: So I have followed these steps and I get the following when trying to run the tests within J-Meter: 

screen shot 2017-02-14 at 17 03 03


Could it be that there is further setup/files required for the apache server?

Comment (maccybee) [inline readme.txt]: Hi,
The test will not necessarily run and generate valid responses; as there wasn't an environment to run it against I built a "theoretical script", as per the info - We do not have a public facing test environment for you to write your script against, so it isn't necessary for this test to be perfectly executable.
Hope that helps?

Comment (Greeff) [inline readme.txt]: Yeah that is fine, from the readme it seemed as though you had mocked an api connection to give back responses as required. 

No worries at all.

Comment (maccybee) [inline readme.txt]: Apologies, I'm not that good :(

**Pull request closed without merging**